libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -37,11 +37,22 @@
             <button
               mat-flat-button
               color="primary"
+              data-testing="add-book"
               (click)="addBookToReadingList(b)" 
               [disabled]="b.isAdded"
+              *ngIf="!b?.isFinished"
               >
               Want to Read
             </button>
+            <button
+              mat-flat-button
+              color="primary"
+              (click)="addBookToReadingList(b)" 
```

Comment:
Why bother adding all these bindings if you can never click the button?

You'd be better off here interpolating the name of the button, e.g. `{{ b?.isFinished? "Finished" : "Want to Read" }}`

Reply:
Good catch. At second glance, I see the only difference b/w these buttons are the labels. Made the appropriate changes